app/controllers/movie_preference_controller.py

```diff
@@ -36,9 +36,15 @@ def update(self, user, movie_preference_id):
     def get(self, user):
         user_movie_preferences = user.movies
 
+        # Wrap get_movie calls inside lambdas, then call these lambdas in parallel.
+        get_movie_lambdas = map(lambda mp: (lambda: get_movie(mp.external_movie_id)), user_movie_preferences)
+        results = call_parallel(get_movie_lambdas)
+
         movies = []
-        for mp in user_movie_preferences:
-            external_movie = get_movie(mp.external_movie_id)
+        for zipped_result in zip(user_movie_preferences, results):
+            (mp, external_movie) = zipped_result
+            if external_movie is None:
+                continue
```

Comment:
What would it mean for this to be `None`? Would that mean the request had failed? If so, would we want to try again instead of failing silently?